k8s/pkg/apis/longhorn/v1beta2/replica.go

```diff
@@ -42,6 +58,19 @@ type ReplicaStatus struct {
 	InstanceStatus `json:""`
 	// +optional
 	EvictionRequested bool `json:"evictionRequested"`
+	// +optional
+	// +nullable
+	RebuildStatus *ReplicaRebuildStatus `json:"rebuildStatus"`
+}
+
+// ReplicaRebuildStatus transiently record the status of replica rebuilding failed
+type ReplicaRebuildStatus struct {
```

Comment:
I just re-checked the original ticket. I think the init idea is that we can use a new field to record multiple kinds of replica failure reasons & messages including rebuilding failure. Can we introduce a more general field like `Condition` or `ErrorInfo` instead?